cognite/client/utils/__init__.py

```diff
@@ -1,6 +1,5 @@
-from cognite.client.utils import (
+from cognite.client.utils import (  # isort: skip
```

Comment:
is this comment for linter?

Reply:
Yeah, the linter wants to remove the imports because it thinks they are unused.